Linkpearl/Plugin.cs

```diff
@@ -91,12 +91,20 @@ public sealed class Plugin : IDalamudPlugin {
 
         var camera = CameraManager.Instance->GetActiveCamera();
         if (camera == null) return null;
+        // gotta cast because it's the CS struct vector3 type
+        var cameraPos = (Vector3) camera->CameraBase.SceneCamera.Object.Position;
 
-        var cameraPos = camera->CameraBase.SceneCamera.Object.Position;
         var cameraViewMatrix = camera->CameraBase.SceneCamera.ViewMatrix;
         var cameraFront = new Vector3(cameraViewMatrix.M13, cameraViewMatrix.M23, cameraViewMatrix.M33);
         var cameraTop = camera->CameraBase.SceneCamera.Vector_1;
         
+        // SoundMicpos is a int between 0-100 that determines how close to the player the sound should be heard from
+        // (0 = at camera, 100 = at player)
+        var soundMicpos = GameConfig.System.TryGetUInt("SoundMicpos", out var m) ? m : 10;
```

Comment:
Is this assumed default of `10` correct? I would personally change this to `0` because that retains the old behaviour of placing the audio listener at the camera.